packages/flutter_tools/lib/src/build_system/targets/ios.dart

```diff
@@ -509,12 +510,32 @@ abstract class IosAssetBundle extends Target {
         .copySync(dsymOutputBinary.path);
     }
 
+    final FlutterProject flutterProject = FlutterProject.fromDirectory(environment.projectDir);
+
+    bool isImpellerEnabled() {
+      if (!flutterProject.ios.infoPlist.existsSync()) {
+        return false;
+      }
+      final Map<String, Object> info = globals.plistParser.parseFile(flutterProject.ios.infoPlist.path);
+
+      final Object? enableImpeller = info['FLTEnableImpeller'];
+
+      if (enableImpeller is bool?) {
```

Comment:
maybe `enableImpeller is bool ? enableImpeller : false`

Reply:
Done. The linter also dropped some wisdom and this ended up at `enableImpeller is bool && enableImpeller`

Reply:
Question marks gone. I guess there's no question about it.